drivers/vulkan/rendering_device_vulkan.cpp

```diff
@@ -2097,49 +2099,54 @@ RID RenderingDeviceVulkan::texture_create_shared(const TextureView &p_view, RID
 	}
 
 	VkImageViewUsageCreateInfo usage_info;
-	usage_info.sType = VK_STRUCTURE_TYPE_IMAGE_VIEW_USAGE_CREATE_INFO;
-	usage_info.pNext = nullptr;
-	if (p_view.format_override != DATA_FORMAT_MAX) {
-		// Need to validate usage with vulkan.
+	if (context->is_device_extension_enabled(VK_KHR_MAINTENANCE_2_EXTENSION_NAME)) {
+		// May need to make VK_KHR_maintenance2 manditory and thus has Vulkan 1.1 be our minimum supported version
```

Comment:
And we need to test this properly on a Vulkan 1.0 device, `VK_STRUCTURE_TYPE_IMAGE_VIEW_USAGE_CREATE_INFO` is Vulkan 1.1 only and only available if the maintenance extension is supported.

Reply:
Seems fine when emulating a Vulkan 1.0 device